src/librustc_borrowck/diagnostics.rs

```diff
@@ -390,6 +390,85 @@ fn foo(a: &mut i32) {
 ```
 "##,
 
+E0505: r##"
+You tried to move out of a value which was borrowed before.
```

Comment:
Sure, I found almost this sentence description for E0507, should I change that as well ?